scripts/processing/06-make_epochs.py

```diff
@@ -115,36 +115,40 @@ def run_epochs(subject_id, tsss=False):
     ica = read_ica(ica_name)
     ica.exclude = []
 
+    filter_label = '-tsss_%d' % tsss if tsss else '_highpass-%sHz' % l_freq
+    ecg_epochs = create_ecg_epochs(raw, tmin=-.3, tmax=.3, preload=False)
+    eog_epochs = create_eog_epochs(raw, tmin=-.5, tmax=.5, preload=False)
+    del raw
+
     n_max_ecg = 3  # use max 3 components
-    ecg_epochs = create_ecg_epochs(raw, tmin=-.3, tmax=.3)
-    ecg_epochs.apply_baseline((None, None))
     ecg_epochs.decimate(5)
+    ecg_epochs.load_data()
+    ecg_epochs.apply_baseline((None, None))
     ecg_inds, scores_ecg = ica.find_bads_ecg(ecg_epochs, method='ctps',
                                              threshold=0.8)
     print('    Found %d ECG indices' % (len(ecg_inds),))
     ica.exclude.extend(ecg_inds[:n_max_ecg])
+    ecg_epochs.average().save(op.join(data_path, '%s%s-ecg-ave.fif'
+                                      % (subject, filter_label)))
+    np.save(op.join(data_path, '%s%s-ecg-scores.npy'
+                    % (subject, filter_label)), scores_ecg)
+    del ecg_epochs
 
     n_max_eog = 3  # use max 2 components
-    eog_epochs = create_eog_epochs(raw, tmin=-.5, tmax=.5)
-    eog_epochs.apply_baseline((None, None))
     eog_epochs.decimate(5)
+    eog_epochs.load_data()
+    eog_epochs.apply_baseline((None, None))
     eog_inds, scores_eog = ica.find_bads_eog(eog_epochs)
     print('    Found %d EOG indices' % (len(eog_inds),))
-    for ep, sco, kind in ([(ecg_epochs, scores_ecg, 'ecg'),
-                           (eog_epochs, scores_eog, 'eog')]):
-        if tsss:
-            ep.average().save(op.join(data_path, '%s-tsss_%d-%s-ave.fif'
-                                      % (subject, tsss, kind)))
-            np.save(op.join(data_path, '%s-tsss_%d-%s-scores.npy'
-                            % (subject, tsss, kind)), sco)
-        else:
-            ep.average().save(op.join(data_path, '%s_highpass-%sHz-%s-ave.fif'
-                                      % (subject, l_freq, kind)))
-            np.save(op.join(data_path, '%s_highpass-%sHz-%s-scores.npy'
-                            % (subject, l_freq, kind)), sco)
```

Comment:
why removing this? @larsoner ok with you?

Reply:
I didn't remove it. It's still there. Just simplified it a bit. See the variable `filter_label`